src/entities/Votes/utils.ts

```diff
@@ -168,16 +170,42 @@ function getNumber(number: number) {
   return Math.floor(number || 0)
 }
 
+export async function getScores(addresses: string[], block?: string | number) {
+  addresses = addresses.map((addr) => addr.toLowerCase())
```

Comment:
I'd avoid the reassignment of the argument.
`const formattedAddresses = addresses.map((addr) => addr.toLowerCase())`